<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: #f6e58d;</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>200</width>
     <height>38</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;이순신 돋움체 M&quot;;
color: #4b7bec;</string>
   </property>
   <property name="text">
    <string>파일 유형</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>371</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="textRbtn">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;이순신 돋움체 M&quot;;</string>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="imageRbtn">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;이순신 돋움체 M&quot;;</string>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="voiceRbtn">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;이순신 돋움체 M&quot;;</string>
      </property>
      <property name="text">
       <string>Voice</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="cameraRbtn">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;이순신 돋움체 M&quot;;</string>
      </property>
      <property name="text">
       <string>Camera</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="micRbtn">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;이순신 돋움체 M&quot;;</string>
      </property>
      <property name="text">
       <string>Mic</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>480</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;이순신 돋움체 M&quot;;
border-radius: 20px;
background-color: #4b7bec;
color:  rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
